spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -527,7 +541,7 @@ def save_as_csv(agg_metric, fname_out, fname_in=None, append=False):
     if not append or not os.path.isfile(fname_out):
         with open(fname_out, 'w') as csvfile:
             # spamwriter = csv.writer(csvfile, delimiter=',')
-            header = ['Timestamp', 'SCT Version', 'Filename', 'Slice (I->S)', 'VertLevel']
+            header = ['Timestamp', 'SCT Version', 'Filename', 'Slice (I->S)', 'VertLevel', 'DistancePMJ']
```

Comment:
Just a quick note: It looks like a couple of tests are [failing](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/2876872450#step:5:322) due to the addition of a new column in the csv output of `aggregate_slicewise.py`. 

Could you update these tests to account for the new column?

Reply:
of course, I am on it 🙂

Reply:
I updated the tests for `test_aggregate_slicewise.py`  in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3429/commits/3e63ff34b4abf2604dfbafdc48a4f8363ca2001c
and `test_batch_processing.py` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3429/commits/3e63ff34b4abf2604dfbafdc48a4f8363ca2001c
The tests still doesn't pass because `batch_processing/catched_results/` needs to be updated so the csv files have the right new format I think (can you confirm @joshuacwnewton ?)

Reply:
Ah, good call! I can rewrite this test so that it's no longer dependent on specific column indexes. I'll open a PR, so feel free to ignore it for now. :smile:

Reply:
great thank you!

Reply:
#3436 has been merged! Feel free to mark this as resolved once this PR is updated with master. :slightly_smiling_face:

Reply:
will do,  thanks for the fix!